src/js/actions/__tests__/staff.spec.js

```diff
@@ -135,4 +142,126 @@ describe('staff action creators', () => {
             });
         });
     });
+
+    describe('fetchSlots', () => {
+        const startDateFormatted = '2017-03-01';
+        const startDate = moment.utc(startDateFormatted);
+        const endDateFormatted = '2017-03-31';
+
+        const availableSlots = {
+            body: [
+                {
+                    available_slots: [
+                        {
+                            date: '2017-03-30',
+                            slots: '2017-03-30T18:00:00Z,2017-03-30T18:30:00Z'
+                        }
+                    ],
+                    id: 4
+                },
+                {
+                    available_slots: [
+                        {
+                            date: '2017-03-31',
+                            slots: '2017-03-31T12:00:00Z,2017-03-31T12:30:00Z'
+                        }
+                    ],
+                    id: 3
+                }
+            ]
+        };
+
+        const allAvailableSlots = {
+            4: {
+                2: {
```

Comment:
Can we comment on what these ids represent?  like `// Service Id` and `// Resource Id`